pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/default.nix

```diff
@@ -0,0 +1,24 @@
+{ webcord
+, substituteAll
+, callPackage
+, lib
+,
+}:
+webcord.overrideAttrs (old: {
+  patches =
+    (old.patches or [ ])
+    ++ [
+      (substituteAll {
+        src = ./add-extension.patch;
+        vencord = callPackage ./vencord-web-extension { };
+      })
+    ];
+  meta = with lib;
+    old.meta
+    // {
+      description = "Webcord with Vencord web extension";
+      license = licenses.gpl3Only;
+      platforms = [ "x86_64-linux" ];
```

Comment:
Why limit to just x86_64-linux?

Reply:
Since one of its dependencies, vencord, is marked as x86_64 for now